core/src/test/java/greencity/service/impl/PlaceCommentServiceImplTest.java

```diff
@@ -0,0 +1,96 @@
+package greencity.service.impl;
+
+import static org.junit.Assert.assertEquals;
```

Comment:
Methods for asserting reside in the org.junit.jupiter.api.Assertions class instead of org.junit.Assert class.